docs/HACKING.md

```diff
@@ -74,7 +81,7 @@ The recommended developement environment uses a prebuilt OpenStack image.  This
 
```

Comment:
The preceding paragraph starts, "The recommended developement environment." 

"development"